src/ui/common/src/components/integrations/connectedIntegrations.tsx

```diff
@@ -111,6 +111,10 @@ export const ConnectedIntegrations: React.FC<ConnectedIntegrationsProps> = ({
         }}
       >
         {[...integrations]
+          .filter(
```

Comment:
Would you mind leaving a comment here explaining that this is a bit of a temporary fix?